packages/design-system/package.json

```diff
@@ -14,6 +14,27 @@
     "test:unit": "cypress run --component",
     "test:coverage": "nyc check-coverage"
   },
+  "dependencies": {
+    "@types/react": "^18.0.27",
+    "typescript": "^4.9.5",
+    "vite-plugin-istanbul": "^4.0.1",
+    "bootstrap": "^5.2.3",
```

Comment:
Thanks for testing it meticulously! I was thinking of assessing all the messages in this issue 
- #99 

Because there are more info messages like this one and I was thinking of creating a react context or something like that where we can throw any message at any point of the application and then the context updates the alert banner. We can keep track of the existing messages by adding comments on that issue so we don't miss any message. What do you think?

I mean, if this info box is important for the demo I can add it now, but I'd prefer to asses all the messages at once in a separate issue